app/src/main/java/rec/games/pokemon/teambuilder/PokeAPIUtils.java

```diff
@@ -105,4 +106,14 @@ static NamedAPIResourceList parsePokemonListJSON(String pokemonListJSON)
 		Gson gson = new Gson();
 		return gson.fromJson(pokemonListJSON, NamedAPIResourceList.class);
 	}
+
+	static String getPokeId(String url)
```

Comment:
Could you refactor the else out?

The following below would be more concise:
```java
if(url != null)
  return stuff;
return "0";
```

```java
if(url == null)
  return "0";
return stuff;
```

Also shouldn't the return type be int instead of String?

Reply:
I will do that. The return type could be an int if you want, although I mostly did it because I thought that the number will never change. If you want I could change it to a int.

Reply:
It probably doesn't matter. But the PokeAPI documentation says that it is guaranteed to be an integer. Which we should probably be consistent with

Reply:
I can do that. Give me a moment to do so.

Reply:
Is this what you meant?